components/Item/Card/VirtualCard.vue

```diff
@@ -0,0 +1,18 @@
+<template>
```

Comment:
What's the point in this?

Reply:
Although in single-file templates mutating the props of the children components work and it's reactive, in this DOM recyling approach we're working with real instances. Thus, we can't mutate props programatically, we can only mutate data (console errors appears and nothing happens).

This component serves as a wrapper of the card one (which takes item as a prop), but it has no props and takes all BaseItemDto using it's data. That way, we keep backwards compatibility for places where we don't need virtual loaders, like item pages, home page, etc...